src/View/TemplatePaths.php

```diff
@@ -258,14 +258,14 @@ public function resolveTemplateFileForControllerAndActionAndFormat($controller,
 	}
 
 	/**
-	 * @param string $controllerName
+	 * @param string|NULL $controllerName
 	 * @param string $format
 	 * @return array
 	 */
 	public function resolveAvailableTemplateFiles($controllerName, $format = self::DEFAULT_FORMAT) {
```

Comment:
better yet, i created a Milestone for 2.0 and added an issue for that change to that milestone :)
https://github.com/TYPO3Fluid/Fluid/issues/177